.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do you need to put something in package.json to reformat? And IDEs just work with this or need something else?

Reply:
I think one could add prettier as a project dependency, but vscode just worked